### PR TITLE
chore(prow/jobs/tikv/pd): add wip  presubmits jobs

### PR DESCRIPTION
**Pull request by wuhuizuo**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

### CODE REVIEW

**Comment by wuhuizuo** [comment]
depend on: https://github.com/PingCAP-QE/ci/pull/1662

File: prow/jobs/tikv/pd/pd-presubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+presubmits:
+  pingcap/tidb:
```
Comment (purelind) [inline prow/jobs/tikv/pd/pd-presubmits.yaml:2]: maybe `tikv/pd`

**Review by purelind — APPROVED** [review]
/lgtm

**Pull request merged**